frontend/app/globals.css

```
@@ -2,131 +2,279 @@
+@media (prefers-color-scheme: dark) {
+  :root {
+    --background: #0a0a0a;
+    --foreground: #ededed;
+  }
+}
 
-  --color-success: #10B981;
-  --color-warning: #F59E0B;
-  --color-error:   #EF4444;
+body {
+  font-family: 'Inter', 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)